src/Test.Utilities.Sarif/FileDiffingUnitTests.cs

```diff
@@ -256,6 +262,20 @@ private void CompareActualToExpected(
                                                      expectedSarifTextDictionary[key],
                                                      out SarifLog actual);
 
+                    if (expectedResultFingerprintKeys != null &&
+                        actual != null &&
+                        actual.Runs[0].Results != null)
+                    {
+                        foreach (string expectedFingerprintKey in expectedResultFingerprintKeys)
+                        {
+                            if (actual.Runs[0].Results.Any(r => r.Fingerprints.ContainsKey(expectedFingerprintKey) == false))
+                            {
+                                filesMissingFingerprints.Add(key);
+                                break;
```

Comment:
>break [](http://example.com/codeflow?start=32&length=5)

add your reporting up here, get rid of this break, describe the missing data.

Reply:
Fixed to have complete set of missing fingerprints.